test/runtests.jl

```diff
@@ -16,6 +16,7 @@ template_options = Dict(
   "AuthorName" => "Test",
   "AuthorEmail" => "test@me.now",
   "AskAdvancedQuestions" => true,
+  "AddAllContributors" => true,
```

Comment:
```suggestion
  "AddAllcontributors" => true,
```
I think this is the error

Reply:
ah typo! Thanks for spotting!